### ISSUE TITLE
Another italian model

### ISSUE THREAD

**Issue by DanBmh** [MozillaItalia/DeepSpeech-Italian-Model#116]
Hi,

 I just wanted to notify you that I did train another Italian model some time ago, which you can find here: https://gitlab.com/Jaco-Assistant/deepspeech-polyglot

I did use about the same amount of training data like you, but got much better accuracy results (0.248 vs 0.399 WER on CV-test), so you might be able to find some ideas for hyperparameter optimization there.

PS: It would be great if you can share a link to your model in the pretrained models collection thread: https://discourse.mozilla.org/t/links-to-pretrained-models/62688

**Comment by Mte90** [comment]
Hi thanks for the sharing!
We will look in to that @nefastosaturo soon.

I will write in that thread in the meantime.

**Comment by nefastosaturo** [comment]
Hello @DanBmh thank you for sharing! I found your work some time ago and is a very nice one! I also see that you want to collaborate with RHASSPY, that will be awesome!

About deepspeech, yes the hyperparameters optimization will be one of the next thing to do. I noticed that you used data augmentation and we will for sure follow that strategy

**Comment by DanBmh** [comment]
Hi, this time I would like to notify you, that my new project _Scribosermo_ is now ready to use.
This might be quite interesting for you, because it works well with a small amount of data, I did need ~280h of Spanish to reach competitive results.

You can find a somewhat longer description in my post here: https://discourse.mozilla.org/t/links-to-pretrained-models/62688/26
But I didn't train an Italian model, so this would be up to you:)

**Comment by DanBmh** [comment]
[Scribosermo](https://gitlab.com/Jaco-Assistant/Scribosermo) now has an Italian model as well, reaching a WER of 11.5% with ~360h traindata. 
Thanks for your Mitads project by the way, it was used to create the language model:)
